### PR TITLE
[TIMOB-17235]: iOS Windows in tabGroup flicker on first open

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-17235

### CODE REVIEW

File: iphone/Classes/TiUITabProxy.m [hunk 1]
```diff
@@ -255,6 +255,11 @@ - (void)openWindow:(NSArray *)args
   [window setIsManaged:YES];
   [window setTab:self];
   [window setParentOrientationController:self];
+
+  TiUIView *view = [window view];
+  TiViewController *controller = (TiViewController *)[window hostingController];
+  [view setFrame:controller.view.bounds];
```
Comment (hansemannn) [inline iphone/Classes/TiUITabProxy.m]: How about `[TiUtils setView:controller.view positionRect:controller.view.bounds]`? It also respects the center, although it's not as important here.

**Review by hansemannn — CHANGES_REQUESTED** [review]
It still flickers randomly (both the label and the color behind the tab bottom-bar). Test-case:
```js
Titanium.UI.setBackgroundColor('#000');
 
// create tab group
var tabGroup = Titanium.UI.createTabGroup();
 
 
var win1 = Titanium.UI.createWindow({  
    title:'Tab 1',
    backgroundColor:'#fff'
});
var tab1 = Titanium.UI.createTab({  
    title:'Tab 1',
    window:win1
});
 
var label1 = Titanium.UI.createLabel({
	text:'I am Window 1',
	textAlign:'center',
});
 
win1.add(label1);
 
//
// create base UI tab and root window
//
var win2 = Titanium.UI.createWindow({  
    title:'Tab 1',
    backgroundColor:'#fff'
});
var tab2 = Titanium.UI.createTab({  
    title:'Tab 2',
    window:win2
});
 
var label2 = Titanium.UI.createLabel({
	text:'I am Window 2',
});
 
win2.add(label2);
 
var win3 = Titanium.UI.createWindow({  
    title:'Tab 3',
    backgroundColor:'#fff'
});
var tab3 = Titanium.UI.createTab({  
    title:'Tab 3',
    window:win3
});
 
var label3 = Titanium.UI.createLabel({
	text:'I am Window 3',
});
 
win3.add(label3);
 
//
// create base UI tab and root window
//
var win4 = Titanium.UI.createWindow({  
    title:'Tab 4',
    backgroundColor:'#fff'
});
var tab4 = Titanium.UI.createTab({  
    title:'Tab 4',
    window:win4
});
 
var label4 = Titanium.UI.createLabel({
	text:'I am Window 4',
});
 
win4.add(label4);
 
 
//
//  add tabs
//
tabGroup.addTab(tab1);  
tabGroup.addTab(tab2);  
tabGroup.addTab(tab3);
tabGroup.addTab(tab4);
 
tabGroup.open();
```

**Comment by vijaysingh-axway** [comment]
@hansemannn  I tried around 20 times on iphone/ipad simulator but unable to reproduce it. I assume you are testing on this PR. If you can share a small video, that would be great. Thanks!

**Review by hansemannn — APPROVED** [review]
Tests passing now, **CR approved**!

**Pull request merged**